tf-modules/archive/reconciliation_report.tf

```diff
@@ -15,8 +15,6 @@ resource "aws_lambda_function" "create_reconciliation_report" {
       CMR_HOST                         = var.cmr_custom_host
       DISTRIBUTION_ENDPOINT            = var.distribution_url
       ES_HOST                          = var.elasticsearch_hostname
```

Comment:
What about ES_HOST?

Reply:
So I removed these two variables in reference to the comment at the very end of packages/api/lambdas/create-reconciliation-report.js, where I also removed them from logging.
I will be writing a ticket to remove ES from all the other terraform files where they're not needed (minus Cloud Metrics) so I thought this should be part of that removal.

Reply:
CUMULUS-3983